std/concurrency.d

```diff
@@ -186,7 +186,8 @@ private
     {
         foreach (i, t1; T)
         {
-            static assert(isFunctionPointer!t1 || isDelegate!t1);
+            static assert(isFunctionPointer!t1 || isDelegate!t1,
+                    "Not all passed T are function pointer or delegates");
```

Comment:
function pointer_s_. Also you have the index of the T that is not a function pointer or delegate, you may as well give that as part of the message.

Reply:
done